data/customScripts.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
This file should be valid json, and just an array is not.

```json
{
  "customScripts": []
}
```

would be more acceptable in case a different json reader is used